app/src/views/private/components/render-template.vue

```diff
@@ -27,140 +27,121 @@ const templateEl = ref<HTMLElement>();
 
 const regex = /({{.*?}})/g;
 
+const getNestedValues = (data: any, path: string) => {
+	const pathParts = path.split('.');
+	let currentData = data;
+
+	for (const part of pathParts) {
+		if (Array.isArray(currentData)) {
+			currentData = flatMap(currentData.map((item) => get(item, part)));
```

Comment:
Can we replace `flatMap` with the native `.flatMap()`? Is it functionally the same?